Classes/TypeConverter/EventToArrayConverter.php

```diff
@@ -52,10 +52,13 @@ public function convertFrom($source, $targetType, array $convertedChildPropertie
 
     /**
      * @param mixed $object
-     * @return array
+     * @return array|mixed
```

Comment:
Nope, it's the _result_ of the `jsonSerialize()` call which is `mixed`: http://php.net/manual/en/class.jsonserializable.php